student.rb

```diff
@@ -9,4 +11,9 @@ def initialize(classroom, age, name = 'unknown', parent_permission: true)
   def play_hooky
     '¯\(ツ)/¯'
   end
+
+  def classroom=(classroom)
+    @classroom = classroom
+    classroom.student.push(self) unless classroom.student.includes?(self)
```

Comment:
You have done great by adding the `belongs-to` side of the student-classroom relationship. You have mistakingly used `student` instead of `students`. Kindly adjust it to match the definition within the classroom class.